apps/api/src/app/auth/auth.controller.ts

```diff
@@ -43,7 +43,9 @@ import {
   SwitchOrganization,
   SwitchOrganizationCommand,
 } from '@novu/application-generic';
+import { ApiCommonResponses } from '../shared/framework/response.decorator';
 
+@ApiCommonResponses()
```

Comment:
This new decorator to describe common API responses is applied to all controllers.